### PR TITLE
[Do not merge][SQL] Update Swagger for maintenance configuration property

### PR DESCRIPTION
**Pull request by hotsnr**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/databases.json [hunk 1]
```diff
@@ -968,6 +974,10 @@
           "description": "The date when database was resumed by user action or database login (ISO8601 format). Null if the database is paused.",
           "type": "string",
           "readOnly": true
+        },
+        "maintenanceConfigurationId": {
```
Comment (majastrz) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/databases.json:978]: >maintenanceConfigurationId [](start = 9, length = 26)

You are adding a new property to an existing API version. Is it possible for existing clients that are not aware of `maintenanceConfigurationId` properties to cause data loss when interacting with an updated backend?

Comment (hotsnr) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/databases.json:978]: On the backend we are going to do nothing if 'null' or nothing is specified. So no, data loss is not going to be possible.

Comment (majastrz) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/databases.json:978]: Sounds good.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Review by markcowl — CHANGES_REQUESTED** [review]
Erros identified in Model Validator and Semantic Checker for examples must be fixed

**Comment by markcowl** [comment]
@hotsnr please fix modeul validation failures, and remove the 'do not merge' in the title if you would like this merged.

**Comment by markcowl** [comment]
@hotsnr Closing for lack of activity.  Please reopen if you would like to make these changes.

**Pull request closed without merging**